src/screencast_keys/ops.py

```diff
@@ -1507,3 +1539,23 @@ def invoke(self, context, event):
         self.draw_handler_add(context)
         context.window_manager.modal_handler_add(self)
         return {'RUNNING_MODAL'}
+
+
+@BlClassRegistry()
+class SK_OT_WaitForInitialize(bpy.types.Operator):
+    bl_idname = "wm.sk_wait_for_initialize"
+    bl_label = "Screencast Keys Wait For Initialize"
+
+    initialization_handler = None
+
+    def execute(self, context):
+        cls = self.__class__
+        cls.initialization_handler = bpy.types.SpaceView3D.draw_handler_add(cls.intialization_callback, (None, None), 'WINDOW', 'POST_PIXEL')
+        print(cls.initialization_handler)
```

Comment:
Please use `debug_print` like [this](https://github.com/nutti/Screencast-Keys/blob/9bf75d109cb3b6c8974adf37a022718e5f1ab846/src/screencast_keys/ops.py#L1130) to output debug print.